packages/editor-ui/src/composables/useCanvasOperations.ts

```diff
@@ -1443,7 +1443,7 @@ export function useCanvasOperations({
 
 			if (
 				nodeSaveData.credentials &&
-				settingsStore.isEnterpriseFeatureEnabled(EnterpriseEditionFeature.Sharing)
```

Comment:
I see 29 other instances of `settingsStore.isEnterpriseFeatureEnabled` in the code. can we fix this in the `settingsStore` instead?

Reply:
@elsmr Can you please give a hand here as you're most familiar with the context of these changes?

Reply:
Yep I'll fix this

Reply:
`isEnterpriseFeatureEnabled` changed from a method to a computed in https://github.com/n8n-io/n8n/pull/10022
My PR (https://github.com/n8n-io/n8n/pull/10112) with changes to `useCanvasOperations` was not up-to-date with this change in master.

Why the `ParameterInputWrapper` didn't fail on Ricardo's branch I'm not sure 😕 , set the settings store state correctly in the test and it works again.